src/components/Carousel/index.jsx

```diff
@@ -0,0 +1,66 @@
+import { Component } from "react";
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import aImage from "../../assets/images/a.jpg";
+import bImage from "../../assets/images/b.jpg";
+import cImage from "../../assets/images/c.jpg";
+import "./style.css";
+
+export default class MusicSlider extends Component {
```

Comment:
above in app.jsx you are usng functional component, here you are using class component, make standard

Reply:
which one is best?